flow/scripts/generate_abstract.tcl

```diff
@@ -25,8 +25,6 @@ if {![info exists standalone] || $standalone} {
 }
 
 
-source scripts/generate_lef.tcl
-#source scripts/generate_libs.tcl
```

Comment:
not sure if it matters, but if you are running this script not in a standalone fashion, this script now does nothing since the `generate_lef.tcl` is up on line 22 inside the `if`

Reply:
Done, updated.